compose/src/main/java/com/bottlerocketstudios/compose/pullrequest/PullRequestScreen.kt

```diff
@@ -9,18 +9,26 @@ import androidx.compose.foundation.lazy.items
 import androidx.compose.material.Divider
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
+import com.bottlerocketstudios.compose.filterdropdown.FilterDropDownMenu
 import com.bottlerocketstudios.compose.resources.Dimens
 import com.bottlerocketstudios.compose.resources.brown_grey
 import com.bottlerocketstudios.compose.util.Preview
+import com.bottlerocketstudios.compose.util.PreviewComposable
 import com.bottlerocketstudios.compose.util.asMutableState
 
 @Composable
 fun PullRequestScreen(state: PullRequestScreenState) {
-    Column() {
+    Column {
+        FilterDropDownMenu(
+            Modifier.padding(start = Dimens.grid_2, top = Dimens.grid_4_5, bottom = Dimens.grid_3),
+            state.selectedText.value,
+            state.selectionList.value,
+            state.onFilterSelectionClicked
+        )
+
         Divider(
-            modifier = Modifier.padding(start = 9.dp, end = 9.dp, top = 24.dp, bottom = 24.dp),
+            modifier = Modifier.padding(start = Dimens.grid_1, end = Dimens.grid_1, bottom = Dimens.grid_3),
             color = brown_grey,
```

Comment:
Could this color be onSurface or onBackground?

Reply:
@BottleRocket-Colin was there a way to change onSurface to this color instead of greyish_brown without changing the entire thing? I was trying to find a way..

Reply:
No, the solution is not to change the color palette, it would be to either add to palette or make UI use what is already in palette.  So if it doesn't look acceptable using onSurface (as it is coded now) perhaps we could add a "divider" color in the same way we added a tertiary.  However, this should be avoided as much as possible as we don't want an ever growing list of colors.  Instead we want designs to limit the number of colors to the palette/style guide.  However this might be a good exception.

Reply:
@BottleRocket-Colin got it. I'll add it as divider like tertiary is set. thanks